eclipse-collections-code-generator/src/main/resources/impl/list/mutable/boxedPrimitiveList.stg

```diff
@@ -0,0 +1,113 @@
+import "copyrightOf.stg"
+
+targetPath() ::= "org/eclipse/collections/impl/list/mutable/primitive"
+
+fileName(primitive) ::= "BoxedMutable<primitive.name>List"
+
+class(primitive) ::= <<
+<body(primitive.type, primitive.name, primitive.wrapperName)>
+>>
+
+body(type, name, wrapperName) ::= <<
+<copyrightOf("The Bank of New York Mellon")>
+
+package org.eclipse.collections.impl.list.mutable.primitive;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.RandomAccess;
+
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.api.list.primitive.Mutable<name>List;
+import org.eclipse.collections.impl.list.mutable.AbstractMutableList;
+import org.eclipse.collections.impl.utility.Iterate;
+
+/**
+ * A boxed view of a {@link Mutable<name>List}.
+ *
+ * This is for scenarios where optimization is not a consideration and therefore no further methods will be optimized.
+ * \<p>
+ * This file was automatically generated from template file boxedPrimitiveList.stg.
+ *
+ * @since 12.0.1
```

Comment:
I'm assuming the real version number will be something like 12.1.0, but let's check with @donraab and @nikhilnanivadekar

Reply:
Changed. Thanks.

Reply:
12.0 still

Reply:
Thanks, Don. So, is the correct version number @Since 2.0.1?

Reply:
Addressed. Thanks.